src/Parser/DeathKnight/Frost/Modules/Talents/Frostscythe.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+/**
+ *A sweeping attack that strikes all enemies in front of you for 135% Frost damage. This attack benefits from Killing Machine. Critical strikes with Frostscythe deal 4 times normal damage.
+ */
+
+
+class Frostscythe extends Analyzer{
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  casts = 0;
+  hits = -1; // need to initialize negative to make sure first cast isn't counted as bad
```

Comment:
Just a bit confusing comment, I assume you used to count bad casts and forgot to change the comment? From what I can see it is to prevent adding an extra "good" cast on first cast event (or finished with no casts)?

Reply:
Maybe I should've changed the comment to say "good" instead of "bad" but yeah, its to prevent the if block in `on_cast` from firing on the first cast of frostscythe